testdata/testdata.go

```diff
@@ -183,7 +183,7 @@ func GenTestData(table *schema.Table) schema.CQTypes {
 			data[i] = uuidArrayColumn
 		case schema.TypeInet:
 			inetColumn := &schema.Inet{}
-			if err := inetColumn.Set("192.0.2.1/24"); err != nil {
+			if err := inetColumn.Set("192.0.2.0/24"); err != nil {
```

Comment:
What's the reason for this change?

Reply:
I guess it's because the bitmask filters out the final `.1`, so it amounts to the same, but if Arrow rejects this type of IP we may need to handle it better

Reply:
I think `192.0.2.1./24` is basically the same as `192.0.2.0/24` after applying the mask.

Reply:
@yevgenypats Right, but was this change necessary for the tests to pass, or was it just some cleanup?

Reply:
maybe Im confusing something about inet vs cidr . but I did need it for a test to pass because our own arrow `inet` converted it to `192.0.2.0/24` but I think this happened just from the original `IPNet.String` conversion so I suspect it should affect anything but maybe I missed something and didn't get to the root cause of this, not sure.